frontend/src/pages/ContributionView.jsx

```
@@ -0,0 +1,106 @@
+
+      {/* Main Content */}
+      <div className="flex">
+        {/* Left Sidebar */}
+        <div className="w-72 border-r border-gray-200 p-4 h-[calc(100vh-100px)] overflow-y-auto">
+          <div className="mb-6">
+            <h3 className="font-medium mb-3">Overview of DSA</h3>
+            <div className="space-y-3">
+              <div className="flex items-center gap-3 text-sm p-2 bg-blue-50 rounded">
+                <div className="w-5 h-5 bg-green-500 rounded-full flex items-center justify-center">
+                  <svg className="w-4 h-4 text-white" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                    <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M5 13l4 4L19 7" />
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)